src/main.tsx

```diff
@@ -16,12 +22,19 @@ ReactDOM.createRoot(document.getElementById("root")!).render(
       <BrowserRouter>
         <Routes>
           <Route path="/" element={<div>Home Route</div>} />
-          <Route path="/registration-confirm" element={<RegistrationConfirmation />} />
-          <Route path="/onboarding/glucose-notifications" element={<GlucoseNotificationPrompt />} />
+          <Route
+            path="/registration-confirm"
+            element={<RegistrationConfirmation />}
+          />
+          <Route
+            path="/onboarding/glucose-notifications"
+            element={<GlucoseNotificationPrompt />}
+          />
           <Route path="/welcome" element={<Welcome />} />
           <Route path="/onboarding" element={<Start />} />
-        </Routes >
-      </BrowserRouter >
-    </ApolloProvider >
-  </React.StrictMode >
+          <Route path="/login" element={<Login />} />
```

Comment:
currently the clicking on the registration link or back button goes to non-existent routes and gives off messages in the console. This is probably out of scope and unnecessary but you could add something like "<Route path="/registration" element={<div>registration</div>} />" and something similar for the back button as placeholders.